src/openedx_companion_auth/README.rst

```diff
@@ -0,0 +1,134 @@
+Open edX auth companion
+=======================
+
+An Openedx auth companion plugin that redirects users from edx-platform to MIT applications
+
+Prerequisites
+-------------
+
+- Run the following commands to install pants locally:
+
+  .. code-block:: shell
+
+     curl --proto '=https' --tlsv1.2 -fsSL https://static.pantsbuild.org/setup/get-pants.sh | bash
+
+     export PATH=$PATH:/root/.local/bin
+
+Building
+--------
+
+- Build a pip-installable package into `dist/`:
+
+  .. code-block:: shell
+
+     pants package ::
+
```

Comment:
I think we can keep these steps standardized with other plugins, They don't mention the pants installation steps. That should be part of the main Readme.